.travis.yml

```diff
@@ -1,5 +1,10 @@
 language: csharp
-sudo: false
+sudo: required
+dist: trusty
```

Comment:
I'll check this in anyways. Once your workaround gets in, that problem will be resolved.